File: app_mvp/src/main/java/com/example/jqorz/test1/mvp/delegate/activity/ActivityMvpDelegateCallback.java
package com.example.jqorz.test1.mvp.delegate.activity;


import com.example.jqorz.test1.mvp.MvpView;
import com.example.jqorz.test1.mvp.delegate.BaseDelegateCallback;
import com.example.jqorz.test1.mvp.presenters.MvpPresenter;

/**
 * Created by jqorz on 2017/8/28.
 * 生命周期回调
 */

public interface ActivityMvpDelegateCallback<P extends MvpPresenter<V>, V extends MvpView>
        extends BaseDelegateCallback<P, V> {

}

File: app_mvp/src/main/java/com/example/jqorz/mvptest/lib_kotlin/delegate/activity/ActivityMvpDelegateCallback.kt
package com.jqorz.tallybook.base.mvp.delegate.activity

import com.jqorz.tallybook.base.mvp.MvpView
import com.jqorz.tallybook.base.mvp.delegate.BaseDelegateCallback
import com.jqorz.tallybook.base.mvp.presenters.MvpPresenter

/**
 * 生命周期回调
 */
interface ActivityMvpDelegateCallback<P : MvpPresenter<V>?, V : MvpView> : BaseDelegateCallback<P, V>